test/unit/browser/System.js

```diff
@@ -83,7 +83,7 @@ suite('System.js', function() {
 
   test('System.set', function() {
     var store = $traceurRuntime.ModuleStore;
-    var polyfills = store.getForTesting('src/runtime/polyfills/polyfills.js');
+    var polyfills = store.get(System.version + '/src/runtime/polyfills/polyfills.js');
```

Comment:
https://github.com/google/traceur-compiler/blob/master/test/runtime.html#L13

It just uses a script[src] to an es6 file. This probably broke a while ago. We should probably remover this html file and only use 1 html test runner.